### ISSUE TITLE
Rotation for colshapes

### ISSUE THREAD

**Issue by javadomidi** [multitheftauto/mtasa-blue#1229]
setElementRotation doesn't work for colshape elements

![mta-screen_2020-01-27_13-13-37](https://user-images.githubusercontent.com/58877361/73164559-68613900-4107-11ea-979e-94cb790d8754.png)

this is a factory in angel pine.
i want to get it under a colshape (not several colshapes)
it has rotation problem and i can't change the colshape rotation

local testCol = createColCuboid(-2063 , -2350 , 30 , 320 , 28 , 6 )
setElementRotation(testCol, 0 , 0 , 111)

plus:  i dont want to use multiple colshapes
i don't want to use some other colshapes (other than Cuboid) because it will not fill the buildings perfectly as i want
so i think there is no more solution unless fixing setElementRotation

**Comment by javadomidi** [comment]
![mta-screen_2020-01-27_13-13-37](https://user-images.githubusercontent.com/58877361/73164304-f12ba500-4106-11ea-94d8-5a6a8c369096.png)
this is a factory in angel pine.
i want to get it under a colshape (not several colshapes)
it has rotation problem and i can't change the colshape rotation

local testCol = createColCuboid(-2063 , -2350 , 30 , 320 , 28 , 6 )
setElementRotation(testCol, 0 , 0 , 111)

**Comment by dmi7ry** [comment]
I guess it's not a bug but just architecture's feature because of performance.
Use `createColPolygon` instead.

**Comment by Pirulax** [comment]
Yeah, its probably performance reasons, especially if u calculate it all rotations.

**Comment by Yamsha75** [comment]
This can now be done using [createColPolygon](https://wiki.multitheftauto.com/wiki/CreateColPolygon)
